File: PHP/TriDu.php
<?php
/**
 * @version PHP
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
namespace PHP;

//echo "The result is "+greatestCard(2, 2);

class TriDu {

    /**
     * Check what it is the greatest card.
     */
    public function greatestCard($A = 0, $B = 0): int {
        $result = $A;

        if ($B > $A) {
            $result = $B;
        }

        return $result;
    }

}

File: GO/TriDu.go
/**
 * @version GO
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 */
package GO

/**
 * Check what it is the greatest card.
 */
func greatestCard(A int, B int) int64 {
	result := A

	if B > A {
		result = B
	}

	return int64(result)
}
